fitbenchmarking/fitbenchmarking
# Extracting analytical Jacobian from cutest

#### Description of Work

Fixes #481 

Adds option to use analytical Jacobians when running minimizers on CUTEst problems.

#### Testing Instructions

1. Check that the option is set correctly and the docs are up to date.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
